Makefile

```diff
@@ -200,6 +201,129 @@ noop =
 SPACE = $(noop) $(noop)
 export FACTOR_ROOTS := .
 
+<<<<<<< HEAD
```

Comment:
That didn't merge nicely... We now have `run` scripts in each impl dir instead of these `xxx_RUNSTEP` vars. Sorry for the mess...